extensions-api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/ScenarioActivity.scala

```diff
@@ -44,17 +44,56 @@ sealed trait ScenarioComment {
 
 object ScenarioComment {
```

Comment:
TBH, I don't know if I get why we have this comments-like hierarchy and the `pl.touk.nussknacker.engine.api.Comment` class. And there are no connection between these two?